kentikapi/client_integration_test.go

```diff
@@ -478,11 +434,90 @@ func TestClient_GetAgent(t *testing.T) {
 				},
 			},
 			expectedResult: &syntheticspb.GetAgentResponse{Agent: newDummyAgentForGRPC()},
+		}, {
+			name:              "retry 2 times till success on code Unavailable",
+			request:           &syntheticspb.GetAgentRequest{Id: "968"},
+			expectedRequestID: "968",
+			responses: []gRPCResponse{
+				newErrorGRPCResponse(codes.Unavailable),
+				newErrorGRPCResponse(codes.Unavailable),
+				{
+					nil,
+					&syntheticspb.GetAgentResponse{Agent: newDummyAgentForGRPC()},
+				},
+			},
+			expectedResult:    &syntheticspb.GetAgentResponse{Agent: newDummyAgentForGRPC()},
+			expectedErrorCode: codes.OK,
```

Comment:
This is the case when error is nil, so its code and msg are not checked. So those 2 lines are misleading, because our API does not return error code at all. Please drop those 2 lines

Reply:
Done